CONTRIBUTING.md

```diff
@@ -41,7 +41,7 @@ GitHub provides additional document on [forking a repository](https://help.githu
 
 
 ## Finding contributions to work on
-Looking at the existing issues is a great way to find something to contribute on. As our projects, by default, use the default GitHub issue labels (enhancement/bug/duplicate/help wanted/invalid/question/wontfix), looking at any ['help wanted'](https://github.com/aws/sagemaker-mxnet-serving-container/labels/help%20wanted) issues is a great place to start.
+Looking at the existing issues is a great way to find something to contribute on. As our projects, by default, use the default GitHub issue labels (enhancement/bug/duplicate/help wanted/invalid/question/wontfix), looking at any ['help wanted'](https://github.com/aws/sagemaker-mxnet-inference-toolkit/labels/help%20wanted) issues is a great place to start.
```

Comment:
our issue labels are actually slightly different than the defaults - most relevant is that we've actually deleted the "help wanted" label that's specifically linked here 😂

Reply:
This is default contributing file and we should probably change them on all repo to be the same  🤷‍♀️